package.nls.json

```diff
@@ -197,7 +197,14 @@
     "cmake-tools.configuration.cmake.options.advanced.statusBarLength.description": "Configures the maximum length of visible text in the Status Bar when set to 'compact' mode.",
     "cmake-tools.configuration.views.cmake.folders.description": "Folders",
     "cmake-tools.configuration.views.cmake.projectStatus.description": "Project Status",
+    "cmake-tools.configuration.views.cmake.pinnedCommandsWelcome.description":{
+        "message": "No CMake commands have been pinned yet.\n[Add Command](command:cmake.pinnedCommands.add)",
+        "comment": [
+            "The text in parentheses () should not be localized or altered in any way. Also the square brackets and parentheses themselves [] () should not be altered or as well. However, the text inside the square brackets [] should be localized."
+        ]
+    }, 
     "cmake-tools.configuration.views.cmake.outline.description": "Project Outline",
+    "cmake-tools.configuration.views.cmake.pinnedCommands.description": "CMake: Pinned Commands",
```

Comment:
We shouldn't have the `CMake` part here. This is covered by the "category" in the package.json. 

See why with this screenshot, this applies to all of the added descriptions. Also, is this meant to be a title? 
![image](https://github.com/microsoft/vscode-cmake-tools/assets/86264750/0de7d71a-9dcc-42d4-9792-f4cec0e29220)

Reply:
that's interesting. I only see one CMake: prefix when i launch the extension. But removed the "CMake:" as suggested to match consistency with the other views in our extension.